.github/workflows/ci.yml

```diff
@@ -75,3 +75,41 @@ jobs:
 
       - name: Run nix develop - Test GHC (by running a testsuite subset)
         run: nix develop -Lv --fallback ghc.nix# -c bash -c "hadrian/build -j --flavour=quickest test --test-root-dirs=testsuite/tests/programs"
+
+  cross compiler smoke test:
+    runs-on: ubuntu-latest
+
+    steps:
```

Comment:
If there's some way to not duplicate the setup steps between jobs, I'm all ears.